src/NUnitFramework/framework/Internal/Execution/TextMessageWriter.cs

```diff
@@ -276,6 +277,7 @@ private void WriteExpectedLine(object expected, Tolerance tolerance)
             Write(Pfx_Expected);
             Write(MsgUtils.FormatValue(expected));
             if (_sameValDiffTypes) {
+            
```

Comment:
Would you mind undoing this blank line? Instead of fixing this in a new commit, if you amend the same commit you introduced it in, that keeps the git blame history clean.